src/coreclr/jit/gentree.cpp

```diff
@@ -19730,10 +19745,18 @@ GenTree* Compiler::gtNewSimdBinOpNode(
                 case TYP_LONG:
                 case TYP_ULONG:
                 {
-                    assert((simdSize == 16) || (simdSize == 32));
+                    assert((simdSize == 16) || (simdSize == 32) || (simdSize == 64));
                     assert(compIsaSupportedDebugOnly(InstructionSet_AVX512DQ_VL));
```

Comment:
Just so that I understand, does `compIsaSupportedDebugOnly(InstructionSet_AVX512DQ_VL)` implicitely mean that `compIsaSupportedDebugOnly(InstructionSet_AVX512DQ)` is also true?

Reply:
Yes.